documentation/src/main/asciidoc/topics/search.adoc

```diff
@@ -0,0 +1,1096 @@
+[[indexing_searching]]
+=  Indexing and Searching
+
+== Overview
+
+{brandname} supports indexing and searching of cache values stored as Java Pojo(s) or objects encoded via link:https://developers.google.com/protocol-buffers/[Protocol Buffers], using a powerful search API which complement its main Map-like API.
```

Comment:
There is a comma splice in the sentence. Also avoid adjectives such as "powerful" because they don't really add meaning and are subjective.

How about something like this: "{brandname} provides a search API that lets you index and search cache values stored as Java POJOs or as objects encoded as Protocol Buffers."

Reply:
>  Also avoid adjectives such as "powerful" because they don't really add meaning and are subjective.

What about "Mighty" ?  :)

Just kidding, this has been in the guide for ages, I don't mind removing it

Reply:
yeah I've seen powerful in the docs before. I should probably grep and kill any remaining instances.

Reply:
![](https://i.imgflip.com/44iwbh.jpg)